source/mongorestore/mongorestore-examples.txt

```diff
@@ -345,14 +345,27 @@ namespace:
        You cannot restore the ``system.buckets.weather.bson`` file
        by itself. Attempting to do so results in an error.
 
+Back Up and Restore a Database at a Specific Point in Time
+----------------------------------------------------------
+
+To back up a database at a specific point in time to an :term:`oplog`
+file, use :option:`mongodump --oplog`. To restore the database at a
+point in time with an oplog file, use :option:`mongorestore
+--oplogReplay`. For more details, see :ref:`backup-restore-oplogreplay`.
+
+``mongodump`` can run in the background, and ``mongorestore`` can
+restore data from the database dump. However, you cannot use
+``mongorestore`` to specify a specific point in time from that dump. For
```

Comment:
Do we need this "However, you cannot use [...]" sentence" if we are no longer branding mongorestore --oplogReplay as point in time restore?

Reply:
Adjusted.